doc/config.rst

```diff
@@ -0,0 +1,53 @@
+Configuration File Formats
+============================
+
+Petitboot is capable of parsing multiple flavors of configuration files for gathering the required information
+for booting the next kernel. GRUB2 and SYSLINUX configuration formats are both supported along with a "native"
+format. Documentation for both the `GRUB2 <https://www.gnu.org/software/grub/manual/grub/grub.html#Configuration>`_ 
```

Comment:
Best to use the backticks to contain just the text there, ie.:

```
 `GRUB2` <https://www.gnu.org/software/grub/manual/grub/grub.html#Configuration>
```

\- for both the GRUB2 and SYSLINUX references.